log/log.go

```diff
@@ -0,0 +1,73 @@
+package log
+
+import (
+	"github.com/Sirupsen/logrus"
+	libmachinelog "github.com/docker/machine/libmachine/log"
```

Comment:
I'm pretty confused by this import.  Maybe I'm missing something, but isn't the module this file is in `github.com/docker/machine/libmachine/log` already?